blueocean-dashboard/src/main/js/creation/bitbucket/BbIcon.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
+        <path fill="#4A4A4A" fillRule="evenodd" d="M12.3936499,4.51477409 C8.71684593,4.50882622 5.73730236,3.87002485 5.73849193,3.08762489 C5.73968151,2.30522492 8.72160423,1.67594015 12.3984082,1.68188802 C16.0752122,1.68766596 19.0547558,2.32646732 19.0535662,3.10903722 C19.0522067,3.89160713 16.070454,4.52055202 12.3936499,4.51477409 M12.3955193,0 C18.118221,0 22.7908686,1.5386293 22.7906987,3.44874564 C22.7906987,3.95193554 21.5428353,11.1665333 21.0478025,14.0276293 C20.8258619,15.31084 17.5100087,17.1920667 12.3980684,17.1920667 L12.3980684,17.1767721 L12.3928002,17.1920667 C7.28068995,17.1920667 3.96483675,15.31084 3.74289619,14.0276293 C3.24786337,11.1665333 2,3.95193554 2,3.44874564 C2,1.53845936 6.67281752,0 12.3955193,0 L12.3955193,0.000339878352 L12.3955193,0 Z M19.8670821,16.8951829 C20.0557146,16.8951829 20.2348305,17.0219576 20.2348305,17.3020173 C20.2348305,17.3314168 20.2319415,17.3609862 20.2265034,17.3898759 C19.8291856,19.5164947 19.5389295,21.0252147 19.4879478,21.2549725 C19.1411019,22.8192626 16.0811771,23.9998301 12.3958761,24 L12.3957062,24 L12.3955363,24 C8.71040523,23.9998301 5.65031049,22.8192626 5.30346463,21.2549725 C5.25248288,21.0253847 4.96222677,19.5164947 4.56490898,17.3898759 C4.55947092,17.3609862 4.55658196,17.3314168 4.55658196,17.3020173 C4.55658196,17.0219576 4.73552791,16.8951829 4.92433033,16.8951829 C5.08220383,16.8951829 5.20897845,17.0070029 5.20897845,17.0070029 C5.20897845,17.0070029 7.76877226,19.0342073 12.3957062,19.0343773 C17.0224702,19.0342073 19.582434,17.0070029 19.582434,17.0070029 C19.582434,17.0070029 19.7090386,16.8951829 19.8670821,16.8951829 Z M12.3955193,14.8742663 C10.5705425,14.8742663 9.09105199,13.3946058 9.09105199,11.569799 C9.09105199,9.74465223 10.5705425,8.2653317 12.3955193,8.2653317 C14.2204961,8.2653317 15.6999865,9.74465223 15.6999865,11.569799 C15.6999865,13.3946058 14.2204961,14.8742663 12.3955193,14.8742663 Z M12.3923244,9.91273907 C11.4777118,9.91273907 10.7362671,10.6541837 10.7362671,11.5687963 C10.7362671,12.483409 11.4777118,13.2248536 12.3923244,13.2248536 C13.3069371,13.2248536 14.0483817,12.483409 14.0483817,11.5687963 C14.0483817,10.6541837 13.3069371,9.91273907 12.3923244,9.91273907 Z" />
```

Comment:
The colours where indeed inconsistent, I changed them so now they match the design